hack/lib/golang.sh

```diff
@@ -594,11 +599,11 @@ kube::golang::build_binaries_for_platform() {
 
   for binary in "${binaries[@]}"; do
     if [[ "${binary}" =~ ".test"$ ]]; then
-      tests+=(${binary})
+      tests+=("${binary}")
     elif kube::golang::is_statically_linked_library "${binary}"; then
-      statics+=(${binary})
+      statics+=("${binary}")
     else
-      nonstatics+=(${binary})
+      nonstatics+=("${binary}")
     fi
   done
```

Comment:
This breaks build on MacOS. See [this PR](https://github.com/kubernetes/kubernetes/pull/75668) for further details.

Reply:
Thanks for the notice.